packs/qualys/actions/list_scans.py

```diff
@@ -0,0 +1,13 @@
+from lib.base import QualysBaseAction
+
+__all__ = [
+    'ListScansAction'
+]
+
+
+class ListScansAction(QualysBaseAction):
+    def run(self, launched_after="", state="", target="",
```

Comment:
The qualysapi methods currently default to an empty string, because their API I think considers that a 'no value' for now, I want to retain control over that behavior so I'm handling it here. agreed it's not ideal